packages/blade/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -105,7 +100,6 @@ const progressBarHeight: Record<NonNullable<ProgressBarCommonProps['size']>, 2 |
 
 const ProgressBar = ({
   accessibilityLabel,
-  contrast = 'low',
   intent,
```

Comment:
we wanted to get rid of intent prop right?

Reply:
No 🤔, intent will still be there right? We want to have colored progressbars also